custom_components/luxtronik/sensor.py

```diff
@@ -300,6 +301,19 @@ async def async_setup_entry(
             unit_of_measurement=ENERGY_KILO_WATT_HOUR,
             entity_category=EntityCategory.DIAGNOSTIC,
         ),
+        LuxtronikSensor(
+            hass,
+            luxtronik,
+            device_info,
+            sensor_key="calculations.Heat_Output",
+            unique_id="current_heat_output",
+            name=f"{text_current_heat_output}",
+            icon="mdi:lightning-bolt-circle",
+            device_class=DEVICE_CLASS_ENERGY,
+            state_class=STATE_CLASS_MEASUREMENT,
+            unit_of_measurement=POWER_WATT,
```

Comment:
Not sure whether this is needed, I think it is the default for `DEVICE_CLASS_POWER`.